pkg/southbound/clientManager.go

```diff
@@ -111,7 +111,7 @@ func (target *Target) ConnectTarget(ctx context.Context, device topocache.Device
 func setCertificate(pathCert string, pathKey string) tls.Certificate {
 	certificate, err := tls.LoadX509KeyPair(pathCert, pathKey)
 	if err != nil {
-		log.Println("could not load client key pair", err)
+		log.Warning("could not load client key pair", err)
```

Comment:
everything related to certs failing to me is an error

Reply:
changed